.travis.yml

```diff
@@ -1,12 +1,23 @@
 language: python
-python:
-  - "3.5"
-  - "3.6"
-  - "3.7"
-  - "3.8"
+matrix:
+  include:
+  - name: "Python 3.5 on Xenial"
+    python: 3.5
+    dist: xenial
+  - name: "Python 3.6 on Bionic"
+    python: 3.6
+    dist: bionic
+  - name: "Python 3.7 on Bionic"
+    python: 3.7
+    dist: bionic
+  - name: "Python 3.8 on Bionic"
+    python: 3.8
+    dist: bionic      
+before_install:
+  - sudo apt-get -y install appstream pkg-config libcairo2-dev gir1.2-gtk-3.0 libgirepository1.0-dev
```

Comment:
I see you added the appstream dependency after a failing test where Gramps couldn't find the resource path. Can you elaborate, what does this do?

Reply:
It seems to be needed for the translation files to be built properly.  Why I do not know, I can credit Google with the finding as some users on Launchpad ran into the same issue.

Reply:
OK, thanks.